drop-storage/src/lib.rs

```diff
@@ -32,9 +32,50 @@ pub struct Storage {
 
 const MIGRATIONS_DIR: Dir = include_dir!("$CARGO_MANIFEST_DIR/migrations");
 
+#[cfg(unix)]
+fn prepare_sqlite_file(path: &str) -> io::Result<OpenFlags> {
```

Comment:
The name is misleading, it doesn't prepare the file in any way, but builds the flags instead. `get_sqlite_flags()` or `build_sqlite_flags()` can be an alternative

Reply:
Um, it creates a file and changes it's permissions. I think `get_sqlite_flags()` would be misleading since it doesn't mention messing around with the file system

Reply:
My bad, I started to comment without reading the body of the function